src/daemon/configfile.c

```diff
@@ -1375,4 +1375,3 @@ int cf_util_get_cdtime (const oconfig_item_t *ci, cdtime_t *ret_value) /* {{{ */
 
 	return (0);
 } /* }}} int cf_util_get_cdtime */
-
```

Comment:
Ah, sorry. I think my editor automatically did that little cleanup and I missed it. I have updated the change.